doc/source/admin/tool_panel.rst

```diff
@@ -1,51 +1,236 @@
+The three or more files described in the previous section (``tool_conf.xml``, one or
+more ``shed_tool_conf.xml`` files, and ``migrated_tools_conf.xml``) are all used
+to load tool panel items (tools, sections, labels, and workflows). A file named
+``integrated_tool_panel.xml`` defines the arrangement for displaying these
+loaded items in your Galaxy tool panel.  If this file does not exist in your
+Galaxy installation directory, will be automatically created and populated
+when you start your Galaxy server. It is initially populated based on the order
+in which the tool panel items are loaded. The items are loaded as each tool
+panel configuration file is parsed and its items are loaded. The order in which
+these configuration files are parsed is the order of the comma-separated list of
+files defined in your ``tool_config_file`` setting in your ``config/galaxy.yml``
+configuration file. The ``migrated_tools_conf.xml`` file is always parsed and
+loaded last.
+
+If you uninstall a repository that contains tools, entries for those tools will
+automatically be removed from the shed-related tool panel config file and the
+``integrated_tool_panel.xml`` file.
 
-If you uninstall a repository that contains tools, entries for those tools will automatically be removed from the shed-related tool panel config file and the integrated_tool_panel.xml file.
+The best approach for managing the ``integrated_tool_panel.xml`` file is to
+allow Galaxy to add or remove entries as manually adding or removing them will
+likely result in undesired behavior. 
+The order of sections and labels can be changed using a ``tool_conf_labels.xml``
+file as described above.
 
-The best approach for managing the new integrated_tool_panel.xml file is to allow Galaxy to add or remove entries as manually adding or removing them will likely result in undesired behavior. Manual changes to the file should simply be moving entries around to produce the desired arrangement of your tool panel.
+Tool conf XML syntax
+~~~~~~~~~~~~~~~~~~~~
 
-Sections & Labels
-~~~~~~~~~~~~~~~~~
-* You can add ``labels="updated"`` to the ``<tool>`` XML element to render a label next to the tool in the tool panel.
-* You can add ``<section>`` tag as a child of``<toolbox>`` to create a section that can contains tools.
+A tool configuration file is an XML file mainly by four tags. 
+
+1. ``toolbox`` is the root tag. It has two optional attributes
+  - ``tool_path`` is used to set the path (relative to Galaxy's root dir)
+    containing the tools.
+  - ``monitor`` is a boolean that decides if Galaxy should monitor
+    changes to the file.
+2. ``tool`` defines a tool. 
```

Comment:
I'd probably move this to last in the list, as tools are normally inside sections.